packages/math-toolbar/src/editor-and-pad.jsx

```diff
@@ -100,6 +100,10 @@ export class EditorAndPad extends React.Component {
     const inputIsDifferent = this.input.mathField.latex() !== nextProps.latex;
     log('[shouldComponentUpdate] ', 'inputIsDifferent: ', inputIsDifferent);
 
+    if (!inputIsDifferent && this.props.keypadMode !== nextProps.keypadMode) {
```

Comment:
test?

Reply:
There's absolutely no tests set up for this component currently.